<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourcesManagementWidget</class>
 <widget class="QWidget" name="ResourcesManagementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="4" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>141</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="KPushButton" name="removeButton">
     <property name="whatsThis">
      <string>Remove resource from your Akonadi server.</string>
     </property>
     <property name="text">
      <string>R&amp;emove</string>
     </property>
     <property name="icon">
      <iconset theme="list-remove">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KPushButton" name="editButton">
     <property name="whatsThis">
      <string>Modify your installed resources</string>
     </property>
     <property name="text">
      <string>&amp;Modify...</string>
     </property>
     <property name="icon">
      <iconset theme="configure">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KPushButton" name="addButton">
     <property name="whatsThis">
      <string>Add resource to your Akonadi server.</string>
     </property>
     <property name="text">
      <string>A&amp;dd...</string>
     </property>
     <property name="icon">
      <iconset theme="list-add">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="KFilterProxySearchLine" name="mFilterAccount"/>
   </item>
   <item row="1" column="0" rowspan="4">
    <widget class="Akonadi::AgentInstanceWidget" name="resourcesList" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KFilterProxySearchLine</class>
   <extends>QWidget</extends>
   <header>kfilterproxysearchline.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>Akonadi::AgentInstanceWidget</class>
   <extends>QWidget</extends>
   <header location="global">akonadi/agentinstancewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
